src/components/notice.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Notice(props: {
+  children?: React.ReactNode;
+  important?: boolean;
+  extraClasses?: string;
+}) {
+  const baseClassNames = `shadow-sm shadow-wai-gray bg-pure-white text-xl font-mono font-bold text-center text-wai-gray border-4 rounded-lg border-wai-gray p-4 justify-self-center`;
+  const importantClassNames = props.important ? `bg-yellow-100 border-yellow-500 text-yellow-700` : ``;
+  
+  const allClassNames = `${baseClassNames} ${props.extraClasses || ""} ${importantClassNames || ""}`;
+
+  return (
+    <div className={allClassNames}>
+      {props.children}
+    </div>
+  );
+}
```

Comment:
pls configure your editor to terminate your files with `\n\n` ie the last line here would be `}\n\n`

Reply:
github shows a big red stop sign that i dont like looking at when you go open the file and scroll down to the end of it



also https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file